helper/update_comp_benchmarks.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Updates all yamls in target benchmark set to match with source."""
+import argparse
+import os
+
+from experiment.benchmark import Benchmark
+
+BENCHMARK_DIR = 'benchmark-sets'
+SOURCE_SET = 'all'
+TARGET_SET = 'comparison'
+
+
+def parse_args() -> argparse.Namespace:
+  """parse arguments"""
+  parser = argparse.ArgumentParser(
+      description=
+      'Updates all benchmark yamls in <target> to match with <source>.')
+  parser.add_argument(
+      '-s',
+      '--source',
+      type=str,
+      default=SOURCE_SET,
+      help='The source benchmark set used to update target set.')
+  parser.add_argument('-t',
+                      '--target',
+                      type=str,
+                      default=TARGET_SET,
+                      help='The target benchmark set to update.')
+
+  return parser.parse_args()
+
+
+def main():
+  """Usage: python3 -m helper.update_comp_benchmarks.
+  Optional args:  [--source src_dir] [--target target_dir]"""
```

Comment:
nit: Could you please put usage at the top of the file for easier access? Thanks!